### DISCUSSION TITLE
Section 8 Transfer Learning: Do I need to know all the functions used in a pre-trained model like these?

### DISCUSSION THREAD

**Discussion by mosaabthemuslim** [mrdbourke/pytorch-deep-learning#982] [Q&A]
![Screenshot from 2024-06-23 19-43-58](https://github.com/mrdbourke/pytorch-deep-learning/assets/127981674/96c6412c-a447-440f-a06c-bfe9deacc940)
Like batchnorm2d and MBconv.
What to do?
thanks
